lib/NormalModule.js

```diff
@@ -660,6 +660,11 @@ class NormalModule extends Module {
 		this.addError(error);
 	}
 
+	/**
+	 * @param {string | Function | RegExp} rule the rule
+	 * @param {string} content the content
+	 * @returns {boolean} a boolean or return value from the function
```

Comment:
May I know what should be the return type here?  Ig that's gonna be a `boolean` or a value returned by `rule()` if it's a function.